graphql_compiler/global_utils.py

```diff
@@ -59,3 +59,17 @@ def is_same_type(left, right):
         return is_same_type(left.of_type, right.of_type)
     else:
         return False
+
+
+def assert_set_equality(set1: Set[Any], set2: Set[Any]) -> None:
```

Comment:
I was tempted to write `Set[_Element]` annotations for this functions, but it doesn't seem to have the desired effect. 

https://stackoverflow.com/questions/40033093/mypy-ensure-that-arguments-have-same-type-without-listing-the-types-explicitly
`